Snyk.Common/SnykExtension.cs

```
@@ -72,9 +72,12 @@ public static string GetIntegrationVersion()
                 }
                 catch (Exception e)
                 {
-                    var logger = LogManager.ForContext<SnykExtension>();
+                    if (LogManager.IsInitialized())
```

Do we want to ignore the Exception if the LogManager is not initialized? Shouldn't we at least report it to the user or write it to standard error or such?

---

Not sure if we should inform a user that the logger is not working for us. I think that if everything else works, then it is more important for a user. And this exception must appear only for test case. If user run extension in VS it always have manifest file and version number.